### PR TITLE
Fix bug in adding async results to set when using a task info

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
Oops

### CODE REVIEW

File: Research/ResearchTests/Navigation Tests/TaskViewModelTests.swift [hunk 1]
```diff
@@ -0,0 +1,155 @@
+//
+//  TaskViewModelTests.swift
+//  ResearchTests_iOS
+//
+//  Created by Shannon Young on 3/22/19.
```
Comment (Erin-Mounts) [inline Research/ResearchTests/Navigation Tests/TaskViewModelTests.swift]: needs license header

**Review by Erin-Mounts — APPROVED** [review]
Looks good. one nit.

**Comment by DanWebster** [comment]
Can confirm from testing that this fixes the bug and now includes the async appended data onto the task result.  Approved on my end!

**Review by DanWebster — APPROVED** [review]
Can confirm from testing that this fixes the bug and now includes the async appended data onto the task result. Approved on my end!

**Comment by DanWebster** [comment]
@Erin-Mounts Can you merge this when you get a chance? I'd like to fold this fix in to our testFlight version. Thanks!

**Pull request merged**